src/flatc.cpp

```diff
@@ -352,6 +383,10 @@ int main(int argc, const char *argv[]) {
         }
       }
 
+      if(!has_grpc_lang_set && grpc_enabled) {
```

Comment:
I think we'd need this. If we remove `has_grpc_lang_set` check then even if we provide the language, this condition would run and will show an error.